src/screens/WritingCardScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import Prose from '../components/Prose'
+import Main, { MainChild } from '../components/Main'
+import Screen from '../components/Screen'
+
+const WritingCardScreen = () => {
+  return (
+    <Screen>
+      <Main>
+        <MainChild center>
+          <Prose textCenter>
+            <h1>Programming Card…</h1>
```

Comment:
How about we add the precinct and ballot style on this screen? Nice confirmation.

Reply:
👍 Love it.